### ISSUE TITLE
Feature request: Toggle rendering

### ISSUE THREAD

**Issue by geirsagberg** [Toqozz/bevy_debug_lines#26]
Thanks for the plugin, really useful!

It would be sweet if there was some way to disable rendering of debug lines, so one could toggle them with a hotkey, e.g.

```rust
fn toggle_debug_lines(mut debug_lines_context: ResMut<DebugLinesContext>, input: Res<Input<KeyCode>>) {
  if input.just_pressed(KeyCode.F1) {
    debug_lines_context.enabled = !debug_linex_context.enabled;
  }
}
```

**Comment by Toqozz** [comment]
This is a cool idea, but since the rendering is per-frame, users can also implement toggles themselves fairly easily by just wrapping the call (just an example from my phone):
```rust
fn draw_line(start: Vector3, end: Vector3) {
    if !debug_enable_lines {
        return;
    }    

    bevy_debugLines.draw_line(start, end);    
}
```

It's still a cool idea, but I'm not sure about implementing it.

**Comment by geirsagberg** [comment]
Thanks for the workaround, that could work, but would require a wrapper for each function. The reason I missed the toggle functionality was because `bevy_rapier` (which seems to use your code) does it this way: https://github.com/dimforge/bevy_rapier/blob/master/src/render/mod.rs#L157

However, looking at the code it should not be too hard to use your code directly and add a toggle, so I think I will give that a shot :)

**Comment by Toqozz** [comment]
I guess the easiest-to-use way of doing this is just to put an `enabled` flag on the resource.  If I'm going this far I'd kind of like to have a default system run that would toggle rendering.  But maybe that's overstepping what a plugin like this should do.

Either way, I'll add some toggle.

**Comment by Toqozz** [comment]
This should be available now in `0.10.1`.  See the `toggle_rendering` example for how to use it: https://github.com/Toqozz/bevy_debug_lines/blob/master/examples/toggle_rendering.rs

Something I didn't consider was that solving this on the user side isn't actually possible with lines of time > 0, my bad.

Enjoy!
